### PR TITLE
Mint workflow

### PR DESCRIPTION
**Pull request by Amirsharifico**

### CODE REVIEW

File: src/pages/home/index.tsx [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+import { CHAIN_NAMESPACES, SafeEventEmitterProvider, WALLET_ADAPTERS } from "@web3auth/base";
+import React, { useEffect, useState } from "react";
+import { SolflareAdapter } from "@web3auth/solflare-adapter";
+import { SlopeAdapter } from "@web3auth/slope-adapter";
+import { SolanaWalletConnectorPlugin } from "@web3auth/solana-wallet-connector-plugin";
+import RPC from "../../utils/solanaRPC";
+import { OpenloginAdapter } from "@web3auth/openlogin-adapter";
+import Api from "../../services/api";
+const clientId =
+  "BEssGOFZDUSOPpzXZvdSQ02nXBQUCA13EQ7pIIdPfnichG55JR8kU0pU5zRGo-xJuza0SUl5Gx9e2PXOuvMz1og"; // get from https://dashboard.web3auth.io
```
Comment (saberistic) [inline src/pages/home/index.tsx:14]: this should be an env

File: src/pages/home/index.tsx [hunk 2]
```diff
@@ -0,0 +1,298 @@
...
+  const [collectionMint, setcollectionMint] = useState<string>("");
+  useEffect(() => {
+    const init = async () => {
+      try {
+        const web3auth = new Web3Auth({
+          clientId,
+          chainConfig: {
+            chainNamespace: CHAIN_NAMESPACES.SOLANA,
+            chainId: "0x3", // Please use 0x1 for Mainnet, 0x2 for Testnet, 0x3 for Devnet
+            rpcTarget: "https://api.devnet.solana.com", // This is the public RPC we have added, please pass on your own endpoint while creating an app
```
Comment (saberistic) [inline src/pages/home/index.tsx:39]: turn to env

File: src/pages/home/index.tsx [hunk 3]
```diff
@@ -0,0 +1,298 @@
...
+          },
+          uiConfig: {
+            theme: 'dark',
+            // loginMethodsOrder:['email_passwordless','google'],
+            appLogo: 'https://global-uploads.webflow.com/61cdf3c5e0b8155f19e0105b/61d33553e398cf5f9cc90bf7_webclip.png'
+          },
+          web3AuthNetwork: "testnet",
```
Comment (saberistic) [inline src/pages/home/index.tsx:46]: another env

File: src/utils/solanaRPC.ts [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+        return error as string;
+      }
+    };
+  
+    createNFT=async():Promise<string> => {
+      try {
+        
+        let privateKey=await this.getPrivateKey();
+        let res="";
+    await fetch('http://localhost:4000/uploadNFT')
```
Comment (saberistic) [inline src/utils/solanaRPC.ts:93]: service address should be env too

File: src/pages/home/index.tsx (hunk 1 above)
Comment (Amirsharifico) [reply src/pages/home/index.tsx:14]: Done

File: src/pages/home/index.tsx (hunk 2 above)
Comment (Amirsharifico) [reply src/pages/home/index.tsx:39]: Done

File: src/pages/home/index.tsx (hunk 3 above)
Comment (Amirsharifico) [reply src/pages/home/index.tsx:46]: done

File: src/utils/solanaRPC.ts (hunk 4 above)
Comment (Amirsharifico) [reply src/utils/solanaRPC.ts:93]: done

**Pull request merged**